content/docs/howto/configuration.md

```diff
@@ -252,6 +252,73 @@ For example, to make the BellSoft Liberica JRE dependency accessible available t
    * A key/value pair where the key is equal to the `sha256` of the dependency and the value is equal to the new URI.
 4. Configure all builds with this binding.
 
+### Dependency Mirrors
+Larger (corporate) networks might have a mirror server available to cache dependencies for access from within the local network. Dependency mirrors can be used to download all (or some) buildpack dependencies from such alternative locations regardless of their versions.
```

Comment:
```suggestion
Larger networks might have a mirror server available to cache dependencies for access from within the local network. Dependency mirrors can be used to download all (or some) buildpack dependencies from such alternative locations regardless of their versions.
```

Even if this is the state of the world most of the time I don't think that it needs to be in the documentation.